src/components/ContentSidebar/ActivityFeed/messages.js

```diff
@@ -0,0 +1,192 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    approvalAddAssignee: {
+        defaultMessage: 'Add an assignee',
+        description: 'Placeholder for approvers input',
+        id: 'boxui.activityFeed.approvalAddAssignee'
+    },
+    approvalAddTask: {
+        defaultMessage: 'Add Task',
+        description: 'Label for checkbox to add approvers to a comment',
+        id: 'boxui.activityFeed.approvalAddTask'
+    },
+    approvalAddTaskTooltip: {
+        defaultMessage:
+            'Assigning a task to someone will send them a notification with the message in the comment box and allow them to approve or deny.',
+        description: 'Tooltip text for checkbox to add approvers to a comment',
+        id: 'boxui.activityFeed.approvalAddTaskTooltip'
+    },
+    approvalAssignees: {
+        defaultMessage: 'Assignees',
+        description: 'Title for assignees input',
+        id: 'boxui.activityFeed.approvalAssignees'
+    },
+    approvalDueDate: {
+        defaultMessage: 'Due Date',
+        description: 'Title for approvers due date input',
+        id: 'boxui.activityFeed.approvalDueDate'
```

Comment:
search/replace boxui ---> be in id prefix.
The key for the messages should also same as id. See the global messages.js.

We may have to move this file and merge it with the global messages.js, since there are build tools around it for publishing.